packages/ui/src/theme/css/component/toggleButtonGroup.scss

```diff
@@ -0,0 +1,22 @@
+.amplify-togglebuttongroup {
+  align-items: var(--amplify-components-togglebuttongroup-align-items);
+  align-content: var(--amplify-components-togglebuttongroup-align-content);
+  justify-content: var(--amplify-components-togglebuttongroup-justify-content);
+  gap: 0;
+}
+
+.amplify-togglebuttongroup {
+  .amplify-togglebutton {
+    &:not(:first-of-type) {
+      border-start-start-radius: 0;
+      border-end-start-radius: 0;
+      &:not(:focus) {
+        border-left: 1px solid transparent;
```

Comment:
Should these be in a variable? And can they use logical properties?

Reply:
I agree on using logical one but this is specific for buttons in a group to avoid border overlap in between. So I think users may not necessarily want to change that. For the focus selector there, just don't want to have one side look thinner than the other three. So it would be:

![Screen Shot 2021-10-01 at 1 50 15 PM](https://user-images.githubusercontent.com/40295569/135684800-567c1369-3de3-43ba-bf65-adbfdc1d86f0.png)

rather than:

![Screen Shot 2021-10-01 at 1 50 36 PM](https://user-images.githubusercontent.com/40295569/135684838-54ea80e9-bf41-4a26-903f-9fe98fab9002.png)

Does that make sense?